hothouse/_version.py

```diff
@@ -22,9 +22,9 @@ def get_keywords():
     # setup.py/versioneer.py will grep for the variable names, so they must
     # each be defined on a line of their own. _version.py will just call
     # get_keywords().
-    git_refnames = "$Format:%d$"
-    git_full = "$Format:%H$"
-    git_date = "$Format:%ci$"
+    git_refnames = " (HEAD -> master)"
```

Comment:
Changes to this file should be reverted -- this file tracks versions and won't need updating.  It looks like you've committed from an installed version, rather than `pip install -e .` and then editing and committing.

Reply:
@cheinemann can you please fix this bit?